Clicker/SectionControllers/PollTypeSectionController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PollTypeSectionController.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 8/27/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+protocol PollTypeSectionControllerDelegate {
+    var editSessionDelegate: EditSessionDelegate {get}
+    func sectionControllerWillDisplayPollType(sectionController:PollTypeSectionController, pollType: PollType)
+}
+
+class PollTypeSectionController: ListSectionController, ListDisplayDelegate {
+    
+    var pollTypeModel: PollTypeModel!
+    var delegate: PollTypeSectionControllerDelegate!
+    
+    init(delegate: PollTypeSectionControllerDelegate) {
+        self.delegate = delegate
+    }
+    
+    // MARK: - ListSectionController overrides
+    override func sizeForItem(at index: Int) -> CGSize {
+        // This is the size of the whole collection view
+        return (collectionContext?.containerSize)!
```

Comment:
can we safely unwrap this. Like `guard let context = collectionContext else { return .zero }`

Reply:
yep will do